app/src/common/app_main.h

```diff
@@ -35,6 +35,11 @@
 #define INS_GET_VERSION                 0
 #define INS_GET_ADDR_ED25519            1
 #define INS_SIGN_ED25519                2
+// #define INS_GET_ADDR_SR25519         3
```

Comment:
can we remove the comments to avoid any future error? Are the final values 4, 5 and 7?